{"uuid": "749178ed-c0d5-48cd-945b-87195161c69a", "children": ["3584ba50-48e2-4b0a-9873-14e5baa5fcbe"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\Desktop\\SW Testing\\Pytest framework\\CredKart_pytest_project 31-07-2023\\CredKart_Pytest_Project - Copy\\testCases\\conftest.py\", line 28, in setup\n    driver = webdriver.edge()\n             ^^^^^^^^^^^^^^^^\n"}, "start": 1691477098681, "stop": 1691477098681}], "start": 1691477098681, "stop": 1691477098683}